get-started/part2.md

```diff
@@ -202,7 +202,7 @@ between what you `EXPOSE` within the `Dockerfile`, and what you `publish` using
 `docker run -p`. In later steps, we'll just map port 80 on the host to port 80
 in the container and use `http://localhost`.
 
-Hit `CTRL+C` in your terminal to quit.
+Hit `CTRL+C` in your terminal to quit STDOUT from the container and run `docker ps` to get the container id to stop it with `docker stop [container id]`.
```

Comment:
I don't think "quitting STDOUT" is a thing. Is there a way to rephrase this to be more accurate?

Reply:
I think this behavior is specific the cmder shell, so I'm not inclined to add this special case to the topic.